userdocs/server/bootloaders.rst

```diff
@@ -391,3 +402,56 @@ The wwinit module provisions to tmpfs. By default, tmpfs is permitted to use up
 to 50% of physical memory. This size limit may be adjusted using the kernel
 argument `wwinit.tmpfs.size`. (This parameter is passed to the `size` option
 during tmpfs mount. See ``tmpfs(5)`` for more details.)
+
+Persisitent installation
+========================
+With the `dracut` installation enabled warewulf can also install 
+the node image to a harddrive. On the first boot of the node the
+compressed node image is simply dumped onto the configured partition.
+Subsequent reboots will update only image with `rsync` except the
+node image has changed. Then the configured partition will be erased and 
+the configured node image will be dumped on the disk.
+
+.. warning::
+   warewulf doesn't install the bootloader to the disk and add UEFI 
+   entries. In order to boot the node network booting is required and 
+   at every boot the kernel and the initrd is transfered over the network.
+
+Configuration
+-------------
+
+As the first step you will have to enable the `rsyncd` (which must be installed) 
+Management in `warewulf.conf`. For this simply add
+
+.. code-block:: yaml
+   rsync:
+     systemd name: rsyncd
+     enabled: true
+
+and configure the service with `wwctl configure rsync` which will
+also generate a appropritate `rsyncd.conf`.
+
+The node image will be installed to the partition called `rootfs`. You 
+can add add a rootfs with e.g. following command 
+
+.. code-block:: shell
+
+   wwctl node set n01 \
+     --diskname /dev/vda --diskwipe \
+     --partname rootfs --partcreate --partnumber 1 \
+     --fsname rootfs --fsformat ext4 --fspath /rootfs
```

Comment:
Doesn't this mean it will be mounted twice at runtime? Once by dracut at `/`, and again by systemd at `/rootfs`?